infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesConfigsMaps.java

```diff
@@ -61,11 +61,16 @@
    *
    * @param configMap config map to create
    * @throws InfrastructureException when any exception occurs
+   * @return created {@link ConfigMap}
    */
-  public void create(ConfigMap configMap) throws InfrastructureException {
+  public ConfigMap create(ConfigMap configMap) throws InfrastructureException {
```

Comment:
Just curious how returned config map differs from original?

Reply:
It will have stuff added by k8s, like `creationTimestamp` and others? I'm not sure though.

Reply:
so what's the reason on this change?